spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -45,7 +48,35 @@
  * >             .addExampleHeaders(ExampleHeaders.of("my-header", "headerVal"));
  * > }
  * }</pre>
+ *
+ * @deprecated Use {@link ServerBuilder#annotatedService()} via {@link ArmeriaServerConfigurator} and
+ *             {@link DocServiceConfigurator}.
+ *             <pre>{@code
+ *             > @Bean
+ *             > public ArmeriaServerConfigurator myService() {
+ *             >     return server -> {
+ *             >         server.annotatedService()
+ *             >               .pathPrefix("/my_service")
+ *             >               .exceptionHandlers(new MyExceptionHandler())
+ *             >               .requestConverters(new MyRequestConverter())
+ *             >               .responseConverters(new MyResponseConverter())
+ *             >               .decorator(LoggingService.newDecorator())
+ *             >               .build(new MyAnnoatedService());
+ *             >     };
+ *             > }
+ *
+ *             > @Bean
+ *             > public DocServiceConfigurator myServiceDoc() {
+ *             >     return docService -> {
+ *             >         docService.exampleRequestForMethod(MyAnnotatedService.class,
+ *             >                                            "myMethod", "{\"foo\":\"bar\"}")
+ *             >                   .exampleHttpHeaders(MyAnnotatedService.class,
+ *             >                                       HttpHeaders.of("my-header", "headerVal")));
+ *
+ *             >     };
+ *             > }}</pre>
```

Comment:
extra `}`?

Reply:
This seems fine. One is for `myServiceDoc() {` and the other is for `{@code`. 😉

Reply:
😅